src/Tools/MicroBuild/publish-blob.ps1

```diff
@@ -34,7 +34,7 @@ try {
         exit 1
     }
 
-    Exec-Console $msbuild "/p:ConfigDir=$configDir /p:ExpectedFeedUrl=$blobFeedUrl /p:AccountKey=$blobFeedKey /p:OutputPath=$configDir PublishBlobAssets.proj"
+    Exec-Console $msbuild "/p:ConfigDir=$configDir /p:ExpectedFeedUrl=$blobFeedUrl /p:AccountKey=$blobFeedKey /p:OutputPath=$configDir /t:Build PublishBlobAssets.proj"
```

Comment:
Why is this needed? That should be implicit ...

Reply:
Indeed, @mmitche why is this required?

Reply:
Because there is an error in the current build targets that reset the default target:

https://github.com/dotnet/buildtools/blob/280705d9d12af2fe0997509474825d2c9cbe7932/src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets#L1

Reply:
This is just a workaround.  https://github.com/dotnet/buildtools/issues/1822